src/plugins/initiatives/createGraph.js

```diff
@@ -31,6 +33,11 @@ function initiativeNode(initiative: Initiative): Node {
   };
 }
 
+export function initiativeWeight(initiative: Initiative): ?NodeWeight {
+  if (!initiative.weight) return;
+  return !initiative.completed ? initiative.weight[0] : initiative.weight[1];
+}
```

Comment:
Why do we want this to be nullable? Why not just require that all initiatives have weights?

Reply:
To me this seems like the most expressive "internal type". A default value exists that is the equivalent of `null`, namely: `[1, 1]`, but it's surprising, adds needless data and 1x multiplications.

For sake of helping the user "do the right thing©" we do require weights to be set in the `InitiativeFile` type.

Or you could say,  I consider `Initiative` to be correct in this regard, vs `InitiativeFile` which requires weights due to being strongly coupled to our community usage patterns.